internal/view/live_view.go

```diff
@@ -91,23 +91,35 @@ func (v *LiveView) ResourceFailed(err error) {
 	v.text.SetText(cowTalk(err.Error(), x+w))
 }
 
+func (*LiveView) linesWithRegions(lines []string, matches fuzzy.Matches) []string {
+	ll := make([]string, len(lines))
+	copy(ll, lines)
+	offsetForLine := make(map[int]int)
+	for i, m := range matches {
+		loc, line := m.MatchedIndexes, ll[m.Index]
+		offset := offsetForLine[m.Index]
+		loc[0] += offset
+		loc[1] += offset
```

Comment:
Why for both 0 and 1 locations use the same offset? Can you please add unit tests here?

Reply:
Hi @slimus,
Thanks for the review!
loc[0] and loc[1] represent the start and end of a match, to be quoted into region tags, therefore the same offset applies to both. The goal of this offset is to take into account that when multiple matches occur in the same line, and when inserting region tags around a match, the start and end index of subsequent matches get offset by the length of the region tags.
Regarding the unit tests, I will add some.

Reply:
Awesome! Thanks for the explanation and unit tests.